src/test/ui/error-codes/E0523.rs

```diff
@@ -0,0 +1,9 @@
+// aux-build:e0523_aux_1.rs
+// aux-build:e0523_aux_2.rs
+
+extern crate e0523_conflict;
```

Comment:
This should be ambiguous, as two different crates have this name. Maybe it just accepts the first one that it finds and doesn't see the second?